test/org/sagebionetworks/bridge/dynamodb/DynamoStudyTest.java

```diff
@@ -53,12 +53,14 @@ public void studyFullySerializesForCaching() throws Exception {
         assertEquals(study.getMaxNumOfParticipants(), node.get("maxNumOfParticipants").asInt());
         assertEquals(study.getStormpathHref(), node.get("stormpathHref").asText());
         assertEquals(study.getPasswordPolicy(), JsonUtils.asEntity(node, "passwordPolicy", PasswordPolicy.class));
-        assertEquals(study.getVerifyEmailTemplate(), JsonUtils.asEntity(node, "verifyEmailTemplate", EmailTemplate.class));
+        assertEquals(study.getVerifyEmailTemplate(), JsonUtils.asEntity(node, "verifyEmailTemplate",
+                EmailTemplate.class));
```

Comment:
Also 120 lines (which this line is definitely longer than). That said, I have no idea how the auto-format got triggered...